engine/src/main/java/org/entando/entando/web/entity/EntityManagerController.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2018-Present Entando Inc. (http://www.entando.com) All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+package org.entando.entando.web.entity;
+
+import com.agiletec.aps.system.services.role.Permission;
+import org.entando.entando.aps.system.services.entity.IEntityManagerService;
+import org.entando.entando.aps.system.services.entity.model.EntityManagerDto;
+import org.entando.entando.web.common.annotation.RestAccessControl;
+import org.entando.entando.web.common.model.PagedMetadata;
+import org.entando.entando.web.common.model.RestListRequest;
+import org.entando.entando.web.common.model.RestResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/entityManagers")
+public class EntityManagerController {
+
+    @Autowired
+    private IEntityManagerService entityManagerService;
+
+    protected IEntityManagerService getEntityManagerService() {
+        return entityManagerService;
+    }
+
+    public void setEntityManagerService(IEntityManagerService entityManagerService) {
+        this.entityManagerService = entityManagerService;
+    }
+
+    @RestAccessControl(permission = Permission.SUPERUSER)
+    @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
+    public ResponseEntity<?> getEntityManagers(RestListRequest requestList) {
+        PagedMetadata<String> result = this.getEntityManagerService().getEntityManagers(requestList);

```

Comment:
We should put some debug logging in these controllers with the payload.

Reply:
Added log (debug) for response